packages/sources/css/src/components/button/src/index.css

```diff
@@ -89,42 +89,59 @@
 
 .vtmn-btn_variant--primary-reversed:not([disabled]):hover {
   background-color: color-mod(var(--vtmn-color_white) alpha(80%));
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_white);
 }
 
 .vtmn-btn_variant--primary-reversed:not([disabled]):active {
   background-color: color-mod(var(--vtmn-color_white) alpha(60%));
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_white);
 }
 
 /* Secondary button */
 .vtmn-btn_variant--secondary {
-  background-color: transparent;
-  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_black);
-  color: var(--vtmn-color_black);
-  fill: var(--vtmn-color_black);
+  background-color: var(--vtmn-color_white);
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_grey-light-2);
+  color: var(--vtmn-color_brand-digital-dark-1);
+  fill: var(--vtmn-color_brand-digital-dark-1);
 }
 
 .vtmn-btn_variant--secondary:not([disabled]):hover {
-  background-color: color-mod(var(--vtmn-color_grey-dark-2) alpha(5%));
+  background-color: #f2f9fc;
 }
```

Comment:
use variables

Reply:
Thanks for your feedback, I added the new color variables for the light blue